### PR TITLE
Update price feeds early if part of a batch

### PR DESCRIPTION
**Pull request by jayantk**
I was looking at the sui pusher's txs and I noticed that it currently only updates a subset of the price feeds in the pushed VAA. This behavior is an interaction between the structure of the sui contract (with the hot potato etc) and the price pusher.

I'm fixing this problem by adding a feature we'll want anyway, which is to piggyback additional price updates on to a push. With accumulators, it's very cheap to add an additional update to an existing price push, so we'll want to take advantage of this by pushing more feeds than just the ones that met the triggering conditions.

This PR adds an extra set of triggering conditions for an early update. If at least one feed meets the normal triggering conditions, we'll push a batch of updates that includes the original update + all feeds meeting the early update conditions. Using this, we can also set early_update.time_difference = 0 in the sui config to solve the specific problem as well.

I haven't been able to run this since testnet is down, but i will make sure it works before merging.

### CODE REVIEW

File: price_pusher/src/evm/command.ts [hunk 1]
```diff
@@ -91,6 +91,11 @@ export default {
       mnemonic,
       pythContractAddress
     );
+    console.log(
+      `Pushing updates from wallet address: ${pythContractFactory
+        .createWeb3PayerProvider()
+        .getAddress()}`
+    );
```
Comment (jayantk) [inline price_pusher/src/evm/command.ts:98]: drive-by for ease of use

**Review by ali-bahjati — APPROVED** [review]
lgtm! :shipit:

File: price_pusher/src/price-config.ts [hunk 2]
```diff
@@ -59,25 +83,25 @@ export function shouldUpdate(
...
   sourceLatestPrice: PriceInfo | undefined,
   targetLatestPrice: PriceInfo | undefined
-): boolean {
+): UpdateCondition {
   const priceId = priceConfig.id;
 
   // There is no price to update the target with.
   if (sourceLatestPrice === undefined) {
-    return false;
+    return UpdateCondition.YES;
```
Comment (jason-hy-li-crypto) [inline price_pusher/src/price-config.ts:91]: is this a bug ?

Comment (ali-behjati) [reply price_pusher/src/price-config.ts:91]: yes unfortunately. fix is in #2730
